imagetagger/imagetagger/users/views.py

```diff
@@ -250,6 +250,16 @@ def view_team(request, team_id):
     export_format_forms = (ExportFormatEditForm(instance=format_instance) for format_instance in export_formats)
     test_imagesets = imagesets.filter(set_tags__name='test').order_by('-public', 'name')
 
+    # add annotated image count to each imageset
+    for imageset in imagesets:
+        images = Image.objects.filter(image_set=imageset)
+        annotations = Annotation.objects.filter(image__in=images,
```

Comment:
I'd normally object to the two copies of this code but it seems like that's the norm in this codebase 🤷‍♂

Reply:
haha yeah i think thats just how django is